vignettes/download_github_comments.Rmd

```diff
@@ -86,32 +96,28 @@ If you would like to retrieve only issues **before** a certain date, set `date_l
 
 
 ```{r Collect issues by date x, eval = FALSE}
```

Comment:
Can you explain what the following two code blocks for? A user should be able to get a basic understanding of what the code is, and then use code comments and code itself to show it in-action. This applies to any code block that you have throughout the notebook.